Test/Tests/Component/CnmPlugin.Tests.ps1

```diff
@@ -72,7 +71,7 @@ function Save-CnmPluginUnitTestReport {
     Copy-Item $FoundRemoteJUnitReports.FullName -Destination $LocalJUnitDir -FromSession $Session
 }
 
-Describe "CNM Plugin" {
+Describe "CNM Plugin" -Tag "Smoke" {
```

Comment:
Not sure if this should be a smoke test. It requires precompiled test binaries present on host. They are not available after running c-a-d though

Reply:
They are.

Reply:
o_O they shouldn't be I think...